lib/absinthe/blueprint/result/object.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Absinthe.Blueprint.Document.Resolution.Object do
+defmodule Absinthe.Blueprint.Result.Object do
```

Comment:
@bruce moved these into the result namespace. It's both more descriptive of what they are, and also more technically correct. Resolution is not necessary to end up with these.